### PR TITLE
Feature/button changes

### PR DESCRIPTION
**Pull request by MGS31**
Finally finished the button placement for add favourites as well as the disable functionality to stop a user from being able to use this feature when a location is in their favourites.

- [x] Should be able to see the favourites button always.
- [x] Should be able to add a favourite to your list of favourites.
- [x] Once added the button should grey out and functionality should be disabled.
- [x] Favourites should still drop down in the list.
- [x] Once deleted the functionality to add a favourite should return.

### CODE REVIEW

File: leafsmart-front-end/src/components/buttons/AddFavourite.js [hunk 1]
```diff
@@ -2,29 +2,65 @@ import { useAuth } from "../../contexts/AuthContext";
...
+
+  const cityArr = checkFavs(cityName, favs);
 
   const { icon } = props;
   return (
-    <div className="w-16 h-16 bg-gradient-to-r from-Independence to-HeliotropeGray text-White flex items-center justify-center rounded-2xl hover:text-Isabelline hover:rounded-lg transition-all duration-300 ease-linear
-    cursor-pointer"><button onClick={handleClick}>{icon}</button></div>
+    <>
+      {cityArr.includes(cityName) ? (
+        <div
```
Comment (b1u3too) [inline leafsmart-front-end/src/components/buttons/AddFavourite.js:48]: This part could we pair/group tidy it a little? 
I'm wondering why the button is wrapped in a div -- could the classNames be applied directly to the button? I tried it as an experiment while I was adjusting the spacing on my other branch and the button continued to work.

File: leafsmart-front-end/src/components/Dashboard.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ const Dashboard = () => {
       <section className="bg-Isabelline py-5 min-h-screen">
         <div className="flex p-8 justify-between">
           <div className="w-[58%]">
-            <div className="flex justify-between">
+            <div className="flex justify-around">
```
Comment (b1u3too) [inline leafsmart-front-end/src/components/Dashboard.js:35]: Can this change actually be reverted? 
Restyling of the spacing was approached slightly differently on the connect-fav-to-search branch, so this'll cause a conflict.

File: leafsmart-front-end/src/styles/main.css [hunk 3]
```diff
@@ -0,0 +1,1247 @@
+@import url('https://fonts.googleapis.com/css2?family=El+Messiri:wght@400;500;600;700&display=swap');
```
Comment (b1u3too) [inline leafsmart-front-end/src/styles/main.css:1]: We should really get this darn thing out of everyone's cache, lol

**Review by b1u3too — APPROVED** [review]
Alright! This is perfectly fine code and the suggestions that I've popped in are absolutely optional thoughts that I think might help us look _20% cooler_. We're already quite cool. :)
it functions exactly like I expected, thank you very much for making this happen, Matt!

File: leafsmart-front-end/src/components/buttons/AddFavourite.js (hunk 1 above)
Comment (MGS31) [reply leafsmart-front-end/src/components/buttons/AddFavourite.js:48]: Adjusted to only use the button className

File: leafsmart-front-end/src/components/Dashboard.js (hunk 2 above)
Comment (MGS31) [reply leafsmart-front-end/src/components/Dashboard.js:35]: Reverted

**Review by rachelpr — APPROVED** [review]
🚀 everything works as you described!

**Pull request merged**